NEWS.md

```diff
@@ -6,6 +6,15 @@
 
 * Ignore `.env`, `.venv`, and `venv` files only when they reference Python
   virtual environments. (#972)
+  
+* Add `envManagement`, `envManagementR` and `envManagementPy` args when writing
+  the `manifest.json`. These args whether Connect should install packages in the
+  package cache. If `envManagement` is `FALSE` then Connect will not perform any
+  package installation and it is the administrators responsibility to ensure the
+  required R/Python packages are available in the runtime environment.
+  This is especially useful if off-host execution is enabled, when the execution
```

Comment:
Say something about the fact that Connect may ignore these values.

Reply:
Added `These values are ignored when Applications.ManifestEnvironmentManagementSelection = false`